### DISCUSSION TITLE
Disable splash screen

### DISCUSSION THREAD

**Discussion by helloeyesimple** [infinitered/ignite#2413] [Q&A]
Hi,

Is there any way to disable default splash screen? I need to create customer splash screen

**Comment by frankcalise** [comment]
Hey @helloeyesimple - you can absolutely customize the splash screen as you normally would in an RN app.

You can also use the generator to help you out once you have the assets from the customer, see the documentation for more info: https://github.com/infinitered/ignite/blob/master/docs/Generators.md#splash-screen-generator
